android/src/main/java/com/applovin/reactnative/AppLovinMAXAdViewManager.java

```diff
@@ -21,200 +15,63 @@
 class AppLovinMAXAdViewManager
         extends SimpleViewManager<AppLovinMAXAdView>
 {
-    // Parent fields
-    private final ReactApplicationContext reactApplicationContext;
-
-    // Maps from the view to the corresponding ad unit id and ad format.
-    // Both must be set before the MaxAdView is created.
-    private final Map<AppLovinMAXAdView, String>      adUnitIdRegistry = new HashMap<>();
-    private final Map<AppLovinMAXAdView, MaxAdFormat> adFormatRegistry = new HashMap<>();
-
-    // Storage for placement and extra parameters if set before the MAAdView is created
-    private final Map<AppLovinMAXAdView, String>  placementRegistry             = new HashMap<>();
-    private final Map<AppLovinMAXAdView, String>  customDataRegistry            = new HashMap<>();
-    private final Map<AppLovinMAXAdView, String>  adaptiveBannerEnabledRegistry = new HashMap<>();
-    private final Map<AppLovinMAXAdView, Boolean> autoRefreshEnabledRegistry    = new HashMap<>();
-
-    public AppLovinMAXAdViewManager(final ReactApplicationContext reactApplicationContext)
-    {
-        this.reactApplicationContext = reactApplicationContext;
-    }
+    public AppLovinMAXAdViewManager(final ReactApplicationContext reactApplicationContext) { }
 
     @Override
-    public @NotNull String getName()
+    public @NotNull
+    String getName()
```

Comment:
Let's sync live re: AppLovin Java code styling file

Reply:
Rearranged for '@NotNull' to come first.